parmed/amber/_amberparm.py

```diff
@@ -2286,6 +2286,28 @@ def box(self, value):
         else:
             self._box = np.array(value).reshape((-1, 6))[0]
 
+    @property
+    def volume(self):
```

Comment:
Wouldn't it be better to put this on `Structure` instead?

Reply:
Done.